mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -196,7 +196,7 @@ in Terraform state, a `terraform destroy` or `terraform apply` that would delete
     `google_bigquery_table.schema`
 
 * `source_format` (Required) - The data format. Supported values are:
-    "CSV", "GOOGLE_SHEETS", "NEWLINE_DELIMITED_JSON", "AVRO", "PARQUET", "ORC",
+    "CSV", "GOOGLE_SHEETS", "NEWLINE_DELIMITED_JSON", "AVRO", "ICEBERG", "PARQUET", "ORC",
```

Comment:
We can update this description to reference https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externaldataconfiguration for currently publicly supported values to avoid a need to update this later